### ISSUE TITLE
Bad cast while attempting to view settings of an unrelated mod

### ISSUE THREAD

**Issue by duane** [nevir/oni-darkness-not-excluded#15]
Wasn't really thinking when I caused the crash so I can't be more specific, sorry. I'll edit/update if I remember more.

Stacktrace:

```
InvalidCastException: Specified cast is not valid.

(wrapper castclass) System.Object.__castclass_with_cache(object,intptr,intptr)
HarmonyLib.Traverse.GetValue[T] () (at <2f5aaaa000594d07b9998f4cff1e9662>:0)
DarknessNotIncluded.PPanelExtensions.GetChildNamed (PeterHan.PLib.UI.PPanel panel, System.String name) (at <ed2907f88f764f47b2fbb3073277d5eb>:0)
DarknessNotIncluded.OptionsEnhancements.OptionsDialogPresets+Patched_OptionsDialog_AddModInfoScreen.Postfix (System.Object __instance, PeterHan.PLib.UI.PDialog __0) (at <ed2907f88f764f47b2fbb3073277d5eb>:0)
(wrapper dynamic-method) PeterHan.PLib.Options.OptionsDialog.PeterHan.PLib.Options.OptionsDialog.AddModInfoScreen_Patch1(PeterHan.PLib.Options.OptionsDialog,PeterHan.PLib.UI.PDialog)
(wrapper dynamic-method) PeterHan.PLib.Options.OptionsDialog.PeterHan.PLib.Options.OptionsDialog.ShowDialog_Patch1(PeterHan.PLib.Options.OptionsDialog)
PeterHan.PLib.Options.POptions.Process (System.UInt32 operation, System.Object args) (at <73a3ed1bcf054ea2bdd5844646c541c9>:0)
PeterHan.PLib.Core.PRemoteComponent.Process (System.UInt32 operation, System.Object args) (at <5747216dd55949c09886421bdafeb593>:0)
PeterHan.PLib.Options.POptions+ModOptionsHandler.ShowDialog (UnityEngine.GameObject _) (at <5747216dd55949c09886421bdafeb593>:0)
PeterHan.PLib.UI.PButton+<>c__DisplayClass15_0.<Build>b__0 () (at <5747216dd55949c09886421bdafeb593>:0)
KButton.SignalClick (KKeyCode btn) (at <36c1a72976474ac69d137f9f2c30c46c>:0)
KButton.OnPointerClick (UnityEngine.EventSystems.PointerEventData eventData) (at <36c1a72976474ac69d137f9f2c30c46c>:0)
UnityEngine.EventSystems.ExecuteEvents.Execute (UnityEngine.EventSystems.IPointerClickHandler handler, UnityEngine.EventSystems.BaseEventData eventData) (at <e9635660f69b4d9d84c085aeda3be353>:0)
UnityEngine.EventSystems.ExecuteEvents.Execute[T] (UnityEngine.GameObject target, UnityEngine.EventSystems.BaseEventData eventData, UnityEngine.EventSystems.ExecuteEvents+EventFunction`1[T1] functor) (at <e9635660f69b4d9d84c085aeda3be353>:0)
UnityEngine.EventSystems.EventSystem:Update()

Build: U46-552078-SD
```

**Comment by nevir** [comment]
No worries - this is enough detail; thanks!

**Comment by nevir** [comment]
Alright, I've just released v1.3.0 which fixes this!
